src/bin/casc/args.rs

```diff
@@ -1,13 +1,21 @@
 use clap::Parser;
 
 #[derive(Parser, Debug)]
-#[clap(author, version, name = "casc", about = "Compile Cascade SELinux policies into CIL", long_about = None)]
+#[clap(
+    author,
+    version,
+    name = "casc",
+    about = "Compile Cascade SELinux policies into CIL",
+    long_about = "Compile Cascade SELinux policies into CIL\nSupported use cases:\n- Combine all policies: This is the default. \"-o <out_filename>\" to specify the output filename.\n- Build one defined system: \"-s <system_name>\"\n- Build multiple defined systems: \"-s <system_names>\"\n- Build all defined systems: \"-s all\""
```

Comment:
1. When I run "man path/to/casc.1", the line splitting doesn't work for me.  Not sure if that's man page formatting or something clap_mangen is doing, but it's all on one line.
2. Can we elaborate on the Combine all policies a bit more?  Like that the output is a monolithic system policy, and that all configuration options are set to defaults
3. I think the "build one" and "build multiple" options can be combined into one here.  A list of systems with one system listed isn't fundamentally different from a list with more.